astra/src/test/java/com/slack/astra/logstore/search/LogIndexSearcherImplTest.java

```diff
@@ -1354,7 +1354,7 @@ public void testDisabledFullTextSearch() {
                         "1", LogMessage.SystemField.TIME_SINCE_EPOCH.fieldName, "1s"))
                 .hits
                 .size())
-        .isEqualTo(3);
+        .isEqualTo(2);
```

Comment:
this should always have been 2. The 3rd is matching on a `message` field which is KEYWORD. But earlier it would match it since `message` was a predefined field TEXT